.circleci/config.yml

```diff
@@ -164,23 +188,38 @@ workflows:
           <<: *release-tags
           requires:
             - make-release
+
   snapshot-bump:
     when:
       not:
         equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
     jobs:
       - prepare-next-version:
           <<: *only-main-branch
+
   danger:
     when:
       not:
         equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
     jobs:
       - revenuecat/danger
+
   weekly-run-workflow:
     when:
       and:
         - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
         - equal: [ "release-train", << pipeline.schedule.name >> ]
     jobs:
       - revenuecat/automatic-bump
+  
+  manual-trigger-bump:
+    when:
+      equal: [ bump, << pipeline.parameters.action >> ]
+    jobs:
+      - revenuecat/automatic-bump
+
+  update-hybrid-common-versions:
+     when:
+       equal: [ upgrade-hybrid-common, << pipeline.parameters.action >> ]
```

Comment:
I might be wrong but I think this is only for manually triggering the workflow. Do we need another one to make it automatic?

Reply:
this will be triggered from `purchases-hybrid-common` via API, same way it works right now in purchases-flutter